### PR TITLE
vimc-4149 Add proxy for private vis tool report

### PR DESCRIPTION
**Pull request by EmmaLRussell**
This branch
- redirects the url `[base]/2020/visualisation-partners` to the published private data vis tool report (requires login). 
- updates the url for the public data vis tool to `[base]/2020/visualisation`
- accepts 'visualization' variation in both cases (requested by Kim in https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-3881)

This branch is deployed to UAT, so this link should work:
https://support.montagu.dide.ic.ac.uk:10443/2020/visualisation-partners

### CODE REVIEW

File: nginx.montagu.conf [hunk 1]
```diff
@@ -38,8 +38,16 @@ server {
     location /resources/ {
     }
 
-    # Eventually the visualisation app, not authenticated
-    location /visualisation {
+    # Private data visualisation app - authenticated with Montagu via OrderlyWeb, redirects to the report on OW
+    rewrite /2020/visualization-pro /2020/visualisation-pro/ last;
+    location /2020/visualisation-pro/ {
+        proxy_pass  http://orderly_web_web:8888/report/internal-2018-interactive-plotting/version/20191114-152933-6918620a/artefacts/index.html?inline=true;
```
Comment (EmmaLRussell) [inline nginx.montagu.conf:44]: You can test this locally by changing this line to something that will exist (ie not any particularly report version) e.g. 
`proxy_pass  http://orderly_web_web:8888/?;`

File: scripts/dev.sh [hunk 2]
```diff
@@ -52,6 +52,7 @@ openssl dhparam -out workspace/dhparam.pem 1024
 docker cp workspace/certificate.pem reverse-proxy:/etc/montagu/proxy/
 docker cp workspace/ssl_key.pem reverse-proxy:/etc/montagu/proxy/
 docker cp workspace/dhparam.pem reverse-proxy:/etc/montagu/proxy/
+docker cp $here/../2020 reverse-proxy:/usr/share/nginx/html
```
Comment (EmmaLRussell) [inline scripts/dev.sh:55]: Added this in to test serving the html page locally. What was your idea for adding in the real public tool data once it's available? Add it to the container manually or add it to the proxy repo to be deployed?

Comment (richfitz) [reply scripts/dev.sh:55]: ooh, adding it to the proxy is an interesting idea

Comment (richfitz) [reply scripts/dev.sh:55]: currently there's a script that adds it after deploy -  see last line here: https://github.com/vimc/montagu/blob/master/ReleaseProcess.md#deploying-to-uat--science

File: nginx.montagu.conf [hunk 3]
```diff
@@ -38,8 +38,15 @@ server {
     location /resources/ {
     }
 
-    # Eventually the visualisation app, not authenticated
-    location /visualisation {
+    # Private data visualisation app - authenticated with Montagu via OrderlyWeb, redirects to the report on OW
+    rewrite /2020/visualization-partners /2020/visualisation-partners last;
```
Comment (richfitz) [inline nginx.montagu.conf:42]: disappointed that we did not go with visualisation-pro

**Pull request merged**